src/components/Newsletter.js

```diff
@@ -96,6 +98,21 @@ export default function Newsletter(props) {
 
   const isOnScreen = useOnScreen(ref, '0px', 0.7)
 
+  const location = useLocation()
+  useEffect(() => {
+    const params = new URLSearchParams(location.search);
+    const indicateurs = params.getAll('indicateur');
+    const isIndicateur = indicateurs.every(e => ['indice_atmo', 'raep', 'vigilance_meteo', 'indice_uv'].includes(e));
+    if (indicateurs.length > 0 && isIndicateur) {
```

Comment:
Une suggestion parce que j’ai eu du mal à comprendre au début.
```suggestion
    const indicateurs = params.getAll('indicateur').filter(i => i in ['indice_atmo', 'raep', 'vigilance_meteo', 'indice_uv']);
    if (indicateurs.length > 0) {
```

Reply:
Je trouve pas forcément plus lisible mais le comportement me plaît mieux en cas de saisie d'un mauvais indicateur parmi d'autres bons, ça traite les bons et jette les mauvais